parsers/main.js

```diff
@@ -9,4 +9,15 @@ module.exports.getParser = function(uniName, callback) {
 		var parser = require('./' + uniName + '.js');
 		return callback(null, parser.convertToCal);		
 	});		
+}
+
+module.exports.getParser2 = function(uniName, callback) {
```

Comment:
...no...allow the user to do one thing so it's simple, not confusing and doesn't clutter the UI.